control_panel_api/tests/test_aws.py

```diff
@@ -1,88 +1,87 @@
 import json
 
 from unittest.case import TestCase
-from unittest.mock import call
+from unittest.mock import MagicMock, call, patch
 
-from control_panel_api import aws
-from control_panel_api.aws import aws_api_client
+from .aws_mock import AWSClientPatcher, aws
 
 
-class AwsTestCase(TestCase):
+class AwsTestCase(AWSClientPatcher, TestCase):
```

Comment:
Personally I would favour a class level `@patch` - it's simpler, cleaner and more obvious.

Reply:
I sort-of agree. A class level patch would not be as clean for the helm patches. But you're right, the `Patcher` classes are not really right. I'll redo them as class patches.

Reply:
👍

Reply:
I have tried, but I cannot get a class level patch to work. It also clutters the test method signatures, adding the patched object as a parameter.